sim/firesim-lib/src/main/cc/bridges/uart.cc

```diff
@@ -92,7 +89,42 @@ uart_t::uart_t(simif_t *sim,
   fcntl(inputfd, F_SETFL, fcntl(inputfd, F_GETFL) | O_NONBLOCK);
 }
 
-uart_t::~uart_t() { close(this->loggingfd); }
+uart_fd_handler::~uart_fd_handler() { close(this->loggingfd); }
```

Comment:
please add `ci:fpga-deploy" just to sanity check things continue to work correctly.